src/progress-bar/ProgressBar.tsx

```diff
@@ -28,14 +30,17 @@ function ProgressBar(props: ProgressBarProps) {
       className={progressBarClassName}
       style={{
         backgroundColor
-      }}>
+      }}
+      data-testid={testid}>
```

Comment:
Let's investigate how we can turn this into a more accessible ProgressBar. Maybe using this role helps and then we can get the element byRole in the tests.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role